### PR TITLE
Added support for Basic Auth

### PR DESCRIPTION
**Pull request by MaorBril**

### CODE REVIEW

**Comment by fehguy** [comment]
To be clear, this only supports basic auth for reading the specs.

File: modules/swagger-codegen-cli/src/main/java/com/wordnik/swagger/codegen/cmd/Generate.java [hunk 1]
```diff
@@ -64,6 +70,11 @@ public void run() {
             input.setAuth(auth);
         }
 
+        if (isNotEmpty(basicAuth)) {
+            String authorization = new String(Base64Coder.encode(basicAuth.getBytes()));
+            input.setAuth("Authorization:" + "Basic "+ authorization);
```
Comment (lanwen) [inline modules/swagger-codegen-cli/src/main/java/com/wordnik/swagger/codegen/cmd/Generate.java]: why you concatenate it twice?

Comment (webron) [inline modules/swagger-codegen-cli/src/main/java/com/wordnik/swagger/codegen/cmd/Generate.java]: twice?

Comment (wing328) [inline modules/swagger-codegen-cli/src/main/java/com/wordnik/swagger/codegen/cmd/Generate.java]: I think what @lanwen suggested is the following (which concatenates the strings once only):
`input.setAuth("Authorization: Basic "+ authorization);`

Comment (webron) [inline modules/swagger-codegen-cli/src/main/java/com/wordnik/swagger/codegen/cmd/Generate.java]: Oh, I wish those were our issues ;)

**Pull request merged**